### ISSUE TITLE
Ohai can't get platform_version on Amazon Linux 2 GA

### ISSUE THREAD

**Issue by repeatedly** [chef/ohai#1207]
## Description

Amazon Linux 2 GA changes `/etc/system-release` format and it breaks Ohai's `platform_version`.

## Ohai Version

Latest

## Platform Version

Amazon Linux 2 GA

## Ohai Output

- On Amazon Linux 2 GA

```
  "platform": "amazon",
  "platform_version": null,
  "platform_family": "rhel",
  "uptime_seconds": 8642,
  "uptime": "2 hours 24 minutes 02 seconds",
  "idletime_seconds": 68119,
  "idletime": "18 hours 55 minutes 19 seconds",
```
- On Amazon Linux 2 Release Candidate

```
 "platform": "amazon",
  "platform_version": "2",
  "platform_family": "rhel",
  "uptime_seconds": 189531,
  "uptime": "2 days 04 hours 38 minutes 51 seconds",
  "idletime_seconds": 3029975,
  "idletime": "35 days 01 hours 39 minutes 35 seconds",
```

You can check the behaviour with following code

```
def get_redhatish_version(contents)
  contents[/Rawhide/i] ? contents[/((\d+) \(Rawhide\))/i, 1].downcase : contents[/release ([\d\.]+)/, 1]
end

# GA: nil
p get_redhatish_version('Amazon Linux 2')
# RC: 2
p get_redhatish_version('Amazon Linux release 2 (2017.12) LTS Release Candidate')
```

**Comment by tas50** [comment]
Thanks for the excellent report. This is definitely broken and I'll get a fix out ASAP.

**Comment by tas50** [comment]
Fixed by #1214. We'll get this released next week
